lesson_archive/4_cat_costume_2.md

```diff
@@ -40,34 +55,76 @@ First, add the below viewport meta element in your html's <head></head>. This is
+.page--contact {
+    background: url(contact.jpg) center center no-repeat;
+}
 
-Use your own discretion on what looks good. You definitely want each of the html sections to span the whole browser width. Also keep in mind you can use max-width and min-width.
+HTML
+<div class="page--home">...</div>
+<div class="page--about">...</div>
+<div class="page--work">...</div>
+<div class="page--contact">...</div>
+```
 
-Remember you can use both `width: 80%;` and `max-width:400px;` together. This would keep the element at 80% it's container's width, but will not ever get bigger than 400px.
+Notice that the only thing that is different about value of each `background` property above is the `url(...)`. We can consolidate the `center center no-repeat` into one class.
+
+```
+CSS:
+.page {
+    background-position: center center;
+    background-repeat: no-repeat;
+}
+
+.page--home     { background-image: url(home.jpg); }
+.page--about    { background-image: url(about.jpg); }
+.page--work     { background-image: url(work.jpg); }
+.page--contact  { background-image: url(contact.jpg) }
+
+HTML:
+<div class="page page--home">...</div>
+<div class="page page--about">...</div>
+<div class="page page--work">...</div>
+<div class="page page--contact">...</div>
+```
+
+Now, the `background-position` and `background-repeat` CSS properties have been crunched into one class called "page". That way, we can change the position and repeat properties of all `.page` elements at once! The primary difference is that you have to add two classes to the HTML elements. One to apply the `.page` CSS properties, and another to apply the more specific `page--` background images.
 
 #### Course Material for Challenge 4
 
 [Take Udacity Responsive Lesson 3](https://classroom.udacity.com/courses/ud893)
 
 In this lesson you will learn about media queries and a little about flexbox.
 
-#### Challenge 4: Use Media Queries to Help with "Breaks"
+#### Challenge 4: Use media queries to make the site responsive
 
 In today's world, we rarely make sites just adaptive. Usually we want the site to look good on ALL device and don't usually target just one particular device and screen size. This means we want our site to be RESPONSIVE.
 
-**STEP 1: Determine widths where site breaks down**
+**STEP 1: Make the site look good at sizes larger than 1200px.
 
-Start resizing your screen (by dragging in the window size) from big to small. When you notice something "break", like 2 boxes sliding under each other or elements overlapping, then make note of the window width. You can see width width in the top right hand corner of your browser window but ONLY when you have developer tools opened.
+It seems like the desktop site was designed for screen sizes of 1200px. Try to replicate the desktop design inside of a media query for widths larger than 1200px.
```

Comment:
I updated these steps to be more mobile-first friendly since the older instructions were desktop-first.